bayesian_cnn_prometheus/config.json

```diff
@@ -14,6 +14,7 @@
     },
     "create_chunks": {
       "is_activated": true,
+      "chunks_number": 50,
```

Comment:
Jeśli bierzemy randomowe chunki to musimy określić ila ma ich być, żeby generator się kiedyś skończył